cfme/tests/physical_infrastructure/test_redfish_physical_server_details.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.physical.provider.redfish import RedfishProvider
+
+pytestmark = [pytest.mark.provider([RedfishProvider], scope="module")]
+
+
+@pytest.fixture(scope="module")
```

Comment:
you need to change this also as we are using provide otherwise it will give fixture error.

Reply:
done 👍